diff --git a/node_modules/@lit-protocol/access-control-conditions/src/lib/canonicalFormatter.js b/node_modules/@lit-protocol/access-control-conditions/src/lib/canonicalFormatter.js
index 8f6dbb5..474067e 100644
--- a/node_modules/@lit-protocol/access-control-conditions/src/lib/canonicalFormatter.js
+++ b/node_modules/@lit-protocol/access-control-conditions/src/lib/canonicalFormatter.js
@@ -58,6 +58,8 @@ const canonicalUnifiedAccessControlConditionFormatter = (cond) => {
                 return (0, exports.canonicalEVMContractConditionFormatter)(cond);
             case 'cosmos':
                 return (0, exports.canonicalCosmosConditionFormatter)(cond);
+            case 'litAction':
+                return (0, exports.canonicalLitActionConditionFormatter)(cond);
             default:
                 throw new constants_1.InvalidAccessControlConditions({
                     info: {
@@ -382,3 +384,30 @@ const canonicalResourceIdFormatter = (resId) => {
 };
 exports.canonicalResourceIdFormatter = canonicalResourceIdFormatter;
 //# sourceMappingURL=canonicalFormatter.js.map
+/**
+ * Canonical Lit Action Condition Formatter
+ */
+const canonicalLitActionConditionFormatter = (cond) => {
+    if (Array.isArray(cond)) {
+        return cond.map((c) => exports.canonicalLitActionConditionFormatter(c));
+    }
+    if ('operator' in cond) {
+        return getOperatorParam(cond);
+    }
+    if ('returnValueTest' in cond) {
+        return {
+            ipfsId: cond.ipfsId,
+            code: cond.code,
+            jsParams: cond.jsParams,
+            returnValueTest: {
+                key: cond.returnValueTest.key,
+                comparator: cond.returnValueTest.comparator,
+                value: cond.returnValueTest.value,
+            },
+        };
+    }
+    throw new constants_1.InvalidAccessControlConditions({
+        info: { cond },
+    }, 'You passed an invalid access control condition');
+};
+exports.canonicalLitActionConditionFormatter = canonicalLitActionConditionFormatter;
diff --git a/node_modules/@lit-protocol/access-control-conditions/src/lib/validator.js b/node_modules/@lit-protocol/access-control-conditions/src/lib/validator.js
index e2bda82..ae967b1 100644
--- a/node_modules/@lit-protocol/access-control-conditions/src/lib/validator.js
+++ b/node_modules/@lit-protocol/access-control-conditions/src/lib/validator.js
@@ -7,6 +7,7 @@ const misc_1 = require("@lit-protocol/misc");
 const SCHEMA_NAME_MAP = {
     cosmos: 'LPACC_ATOM',
     evmBasic: 'LPACC_EVM_BASIC',
+    litAction: 'LPACC_LIT_ACTION',
     evmContract: 'LPACC_EVM_CONTRACT',
     solRpc: 'LPACC_SOL',
 };
@@ -103,6 +104,9 @@ const validateUnifiedAccessControlConditionsSchema = async (accs) => {
             case 'evmBasic':
                 schema = await getSchema('evmBasic');
                 break;
+            case 'litAction':
+                schema = await getSchema('litAction');
+                break;
             case 'evmContract':
                 schema = await getSchema('evmContract');
                 break;
